generator/regions/region_base.cpp

```diff
@@ -4,11 +4,69 @@
 
 #include "base/assert.hpp"
 #include "base/control_flow.hpp"
+#include "base/string_utils.hpp"
 
 namespace generator
 {
 namespace regions
 {
+std::vector<std::string> RegionWithName::m_preferredLanguages =
```

Comment:
Why do you need this as a class member? Is it not enough to store the vector in an anonymous namespace in .cpp?

Reply:
fixed